client/galaxy/scripts/mvc/collection/collection-model.js

```diff
@@ -130,9 +130,10 @@ var DatasetDCE = DATASET_MODEL.DatasetAssociation.extend(
             ),
 
             _downloadQueryParameters: function() {
-                return `?to_ext=${this.get("file_ext")}&hdca_id=${this.get(
-                    "parent_hdca_id"
-                )}&element_identifier=${this.get("element_identifier")}`;
+                var fileExt = this.get("file_ext"),
+                    elementIdentifier = this.get("element_identifier"),
```

Comment:
Instead of multiple var declarations in one statement, we've split this out elsewhere.  It's easier to see/catch bugs in the more explicit form.

```
var fileExt = blah;
var elementIdentifier = blah2;
```